src/ripple/app/ledger/LedgerTiming.h

```diff
@@ -130,6 +130,14 @@ auto constexpr AV_STUCK_CONSENSUS_PCT = 95;
 
 auto constexpr AV_CT_CONSENSUS_PCT = 75;
 
+// The minimum amount of time to consider the previous round
+// to have taken. This ensures that there is an opportunity
+// for a round at each avalanche threshold even if the
+// previous consensus was very fast. This should be at least
+// twice the interval between proposals (0.7) divided by
+// the interval between mid and late consensus ([85-50]/100).
```

Comment:
See the code above:
```
// percentage of previous close time before we advance
auto constexpr AV_MID_CONSENSUS_TIME = 50;

// percentage of previous close time before we advance
auto constexpr AV_LATE_CONSENSUS_TIME = 85;
```
Mid consensus starts at 50%, late consensus starts at 85%. The interval between 85% and 50% is (85-50)/100.